collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -695,6 +697,14 @@ protected void onActivityResult(int requestCode, int resultCode, final Intent in
                 // file
                 ImageConverter.execute(Collect.TMPFILE_PATH, getWidgetWaitingForBinaryData(), this);
                 File fi = new File(Collect.TMPFILE_PATH);
+
+                //revoke permissions granted to this file due it's possible usage in
```

Comment:
Oops, hadn't noticed this before. "due it's possible" -> "due to its possible usage"